core/app/models/spree/line_item.rb

```diff
@@ -88,12 +87,11 @@ def pre_tax_amount
     alias display_total display_amount
     deprecate display_total: :display_amount, deprecator: Spree::Deprecation
 
-    # Sets price and currency from a `Spree::Money` object
+    # Sets price from a `Spree::Money` object
     #
-    # @param [Spree::Money] money - the money object to obtain price and currency from
+    # @param [Spree::Money] money - the money object to obtain price from
     def money_price=(money)
```

Comment:
Yes, this is good. I have a feeling one could golf around in this method a bit more, but I'm fine leaving it as is because it does the right things.